tests/test_syntax.py

```diff
@@ -50,7 +50,12 @@ def compare_results(cfg, testfile, update=False):
     if 'css' in cfg and len(cfg['css']):
         wrapper = WRAPPER % '\n'.join([CSS_LINK % css for css in cfg['css']])
 
-    check_markdown(testfile, extension, extension_config, wrapper, update)
```

Comment:
Unfortunately, this is not what I was looking for.  I was looking for a simple test in `test_targeted`:

```python
class TestSnippets(unitttest.Testcase):
    "Targeted tests for Snippets."""

    def test_bad_file(self):
        """Test bad file."""
        with self.assertRaises(FileNotFoundException):
            markdown.Markdown(
                extensions=['pymdownx.snippets'],
                extension_configs={'pymdownx.snippets': {'check_paths': True}}
            ).convert('--8<--- "bad.file"')

```

test_syntax is meant to compare actual returned syntax.  It just doesn't make sense to alter the syntax test class to handle a one off case that can be handled with two lines of code.  Now keep in mind I through up the code in this example without testing it, but something very much like it should be added.

Please back out the changes to syntax tests.  And I apologize for maybe not giving the clearer directions.